src/acl.c

```diff
@@ -1168,9 +1168,9 @@ int ACLSetSelector(aclSelector *selector, const char* op, size_t oplen) {
  *              "nopass" status. After "resetpass" the user has no associated
  *              passwords and there is no way to authenticate without adding
  *              some password (or setting it as "nopass" later).
- * reset        Performs the following actions: resetpass, resetkeys, off,
- *              -@all. The user returns to the same state it has immediately
- *              after its creation.
+ * reset        Performs the following actions: resetpass, resetkeys, resetchannels,
+ *              allchannels (if acl-pubsub-default is set), off, clearselectors, -@all.
+ *              The user returns to the same state it has immediately after its creation.
```

Comment:
that comment is a mirror of the text in redis.conf, i see the documentation there is outdated too, let's update it too.

the alternative is to simplify both in a way that they don't require an update.
i.e. just say:
> The user returns to the same state it has immediately after its creation

let's keep the explicit approach for now..

Reply:
right, updated redis.conf